samples/ballerina-by-example/examples/mutual-ssl/mutual-ssl.bal

```diff
@@ -0,0 +1,54 @@
+import ballerina.net.http;
+import ballerina.lang.system;
+
+@http:configuration {
+    basePath:"/hello",
+    //Defines a https port.
+    httpsPort:9095,
+    keyStoreFile:"${ballerina.home}/bre/security/wso2carbon.jks",
+    keyStorePassword:"wso2carbon",
+    certPassword:"wso2carbon",
+    //To enable mutual ssl, have to configure the server by giving verifyClient = "require".
+    sslVerifyClient:"require",
+    //Set trustStore location.
+    trustStoreFile:"${ballerina.home}/bre/security/client-truststore.jks",
+    //Set trustStore password
+    trustStorePassword:"wso2carbon"
+}
+
+service<http> helloWorld {
+    @http:resourceConfig {
+    //Send a get request to the specified endpoint.
+        methods:["GET"],
+        path:"/"
+    }
+
+    resource sayHello (http:Request req, http:Response res) {
+        //Set response payload
+        res.setStringPayload(res, "Successful");
+        //Send response to client
+        res.send(res);
+    }
+}
+
+@doc:Description {value:"Ballerina client connector can be used to connect to the created https server. You have to run the service before running this main function. As this is a mutual ssl connection, client also needs to provide keyStoreFile, keyStorePassword, trustStoreFile and trustStorePassword."}
+function main (string[] args) {
+    //create a client connector providing the uri and the struct containing relevant files and passwords.
+    http:ClientConnector clientConnector = create http:ClientConnector("https://localhost:9095", getConnectorConfigs());
+    //creates a request
+    http:Request req = {};
+    http:Response resp = clientConnector.get("/hello/", req);
+    system:println("Response code: " + resp.getStatusCode(resp));
+    system:println("Response: " + resp.getStringPayload(resp));
+}
+
+function getConnectorConfigs() (http:Options) {
+    http:Options option = {ssl: {keyStoreFile:"${ballerina.home}/bre/security/wso2carbon.jks",
+                                 keyStorePassword:"wso2carbon",
+                                 trustStoreFile:"${ballerina.home}/bre/security/client-truststore.jks",
+                                 trustStorePassword:"wso2carbon"
+                                },
+                           followRedirects: {}
+                          };
+    return option;
+}
```

Comment:
Please make sure source and comment lines are not greater than 85. Otherwise it will be out of the frame in the generated html file

Reply:
Done